datapoint/datapoint.go

```diff
@@ -96,15 +96,15 @@ const (
 	tsProperty metadata = iota
 )
 
-//SetProperty sets a property to be used when the time series associated with the datapoint is created
+// SetProperty sets a property to be used when the time series associated with the datapoint is created
 func (dp *Datapoint) SetProperty(key string, value interface{}) {
```

Comment:
i'd even remove the property getter and setter. If they want to use meta, that's cool, but let's remove the idea of a datapoint with Properties as an entity from code not existing outside of signalfx.

Reply:
ok, nm, i am wrong. we need this internally.